docker/README.md

```diff
@@ -327,6 +327,28 @@ You should now be able to configure [Jetpack Backup & Scan](https://jetpack.com/
 - Server password: `wordpress`
 - WordPress installation path: `/var/www/html`
 
+## Custom plugins & themes in the container
+
+Jetpack Docker environment can be wonderful for developing your own plugins and themes, too.
+
+Since everything under `mu-plugins` and `wordpress/wp-content` are git-ignored, you'll want to keep those folders outside Jetpack repository folder and link them as volumes to your Docker instance.
+
+1. First ensure your containers are stopped (`yarn docker:stop`).
+1. Create a docker-compose file. You can place it anywhere in your computer:
```

Comment:
Maybe it should be `2.` here?

Reply:
Sure, just handy to add all as `1.`s since in Markdown this gets rendered as "1. 2. 3." anyway; avoids annoyance of re-organising indexing when re-organising large lists.

This is just 3 items so I'll change it. :-)

Reply:
That's a smart trick! Feel free to leave it as it is.